src/main/java/io/debezium/connector/yugabytedb/SourceInfo.java

```diff
@@ -110,24 +119,25 @@ protected SourceInfo(YugabyteDBConnectorConfig connectorConfig, OpId lastCommitL
     /**
      * Updates the source with information about a particular received or read event.
      *
+     * @param tabletId Tablet ID of the partition
      * @param lsn the position in the server WAL for a particular event; may be null indicating that this information is not
      * available
      * @param commitTime the commit time of the transaction that generated the event;
      * may be null indicating that this information is not available
      * @param txId the ID of the transaction that generated the transaction; may be null if this information is not available
      * @param tableId the table that should be included in the source info; may be null
      * @param xmin the xmin of the slot, may be null
+     * @param recordTime Hybrid Time Stamp Time of the statement within the transaction.
      * @return this instance
      */
-    protected SourceInfo update(String tabletId, OpId lsn, Instant commitTime, String txId,
-                                TableId tableId,
-                                Long xmin) {
+    protected SourceInfo update(String tabletId, OpId lsn, long commitTime, String txId,
+                                TableId tableId, Long xmin, Long recordTime) {
         this.lsn = lsn;
-        if (commitTime != null) {
-            this.timestamp = commitTime;
```

Comment:
`timestamp` is not set anymore. Is it being used anywhere?

Reply:
It's not being used anywhere, but it gets published as a part of the message in the topic. I can we can still populate the field.